quipucords/api/scanjob/tests_scanjob.py

```diff
@@ -51,40 +51,48 @@ def create(self, data):
                                 json.dumps(data),
                                 'application/json')
 
-    def create_expect_400(self, data):
+    def create_expect_400(self, data, expected_response):
         """We will do a lot of create tests that expect HTTP 400s."""
         response = self.create(data)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        response_json = response.json()
+        self.assertEqual(response_json, expected_response)
 
     def create_expect_201(self, data):
         """Create a source, return the response as a dict."""
         response = self.create(data)
+        response_json = response.json()
+        if response.status_code != status.HTTP_201_CREATED:
+            print('Cause of failure: ')
```

Comment:
Is this just for debugging?

Reply:
Yes.  When test failed unexpectedly (not a 201), print the reason. Helpful for debugging.